styles/Recipe.css

```diff
@@ -0,0 +1,67 @@
+@import './MainMenu.css';
+@import './GlobalStyles.css';
```

Comment:
You import /GlobalStyles.css in MainMenu.css, is it necessary to import it again ?

Reply:
Yeah I agree, but it shouldn't make any consequences.

Reply:
Yes, you are right - I tested GlobalStyles.css first on MainManu and forgot to remove it from here